app/org/sagebionetworks/bridge/hibernate/HibernateAccountDao.java

```diff
@@ -315,44 +303,43 @@ protected String generateHealthCode() {
     @Override
     public String createAccount(Study study, Account account) {
         String userId = BridgeUtils.generateGuid();
-        HibernateAccount hibernateAccount = marshallAccount(account);
-        hibernateAccount.setId(userId);
-        hibernateAccount.setStudyId(study.getIdentifier());
-        hibernateAccount.setCreatedOn(DateUtils.getCurrentMillisFromEpoch());
-        hibernateAccount.setModifiedOn(DateUtils.getCurrentMillisFromEpoch());
-        hibernateAccount.setPasswordModifiedOn(DateUtils.getCurrentMillisFromEpoch());
-        hibernateAccount.setMigrationVersion(AccountDao.MIGRATION_VERSION);
+        account.setId(userId);
+        account.setStudyId(study.getIdentifier());
+        DateTime timestamp = DateUtils.getCurrentDateTime();
+        account.setCreatedOn(timestamp);
+        account.setModifiedOn(timestamp);
+        account.setPasswordModifiedOn(DateUtils.getCurrentDateTime());
```

Comment:
Minor: Any particular reason we use the timestamp variable for two of these setters, but call DateUtils.getCurrentDateTime() for the third?

Reply:
That's a mistake, using timestamp now